CHANGELOG.md

```diff
@@ -4,8 +4,13 @@ Inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 Note: This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.1.0] - 2020-05-22
+### Added
+- Add if_truthy formatter
+
 ## [1.0.0] - 2020-05-18
 ### Added
```

Comment:
This wasn't caught in mike's work, but could you fix this to be `### Changed`?

Reply:
Done in 903b1380ba599f9efb695e552fd5ddb99baec362